core/src/actions/properties.ts

```diff
@@ -340,7 +340,7 @@ export class PropertyProfilePreview extends AuthenticatedAction {
 
     apiData.properties[property.key] = {
       id: property.id,
-      state: property.state,
+      state: null,
```

Comment:
Why was this changed?

Reply:
The type for this object is `ProfilePropertyType` and it should actually be referring to the `ProfileProperty`'s state, but here it was being set to the `Property`'s state. Previously the types were compatible, but since we added a `deleted` state to `Property`, typescript now gives an error here. I actually fixed it previously doing by accepting Property state there too (see https://github.com/grouparoo/grouparoo/pull/1773/commits/bb25e02533c8a564561ec4528c77ca780cc5ac01#diff-20d91e3e2a7bfa969dce006af2bd727a09d0d97f85fff330ac30e936d0ffa727L18-L19) but I didn't think that was the right thing and it doesnt seem like that state isn't really being used.